src/parser/jobs/rdm/index.js

```diff
@@ -30,6 +30,11 @@ export default new Meta({
 	],
 
 	changelog: [
+		{
+			date: new Date('2020-09-29'),
+			Changes: () => <>Changed Severity ratings for Missed Dualcasts per Meru 1 missed is now medium, 2 missed is now major.</>,
```

Comment:
I don't think users care who suggested what, the changelog just says *what* changed. IMO anyway.

Reply:
Fair enough I'll remove that piece, it was on my mind when I typed it which is why it appeared there heh.